src/Interfaces/LazySet.jl

```diff
@@ -251,6 +252,30 @@ Alias for the support vector σ.
 """
 const support_vector = σ
 
+"""
+    isboundedtype(S::LazySet)
+
+Determine whether a set is bounded only based on type information.
+
+### Input
+
+- `S` -- set
+
+### Output
+
+`true` if the set is bounded based on type information.
+Note that some sets may still represent an unbounded set even though their type
+actually (example: [`HPolytope`](@ref)).
+
+### Notes
+
+By default this function returns `false`.
+All set types that can determine boundedness should override this behavior.
+"""
+function isboundedtype(S::LazySet)
```

Comment:
i would expect that `isboundedype` acts on types. we could, *maybe*, allow it to receive instances since there is (almost) no room for confusion, but in the first place, it should work types.

Reply:
(correspondingly, `isbounded` should work on instances in the first place..)

Reply:
This is not what you proposed in the issue, which links [here](https://github.com/JuliaReach/LazySets.jl/issues/1914#issuecomment-574742790). The motivation was that the result of `P = HPolytope()` depends on either the type (a polytope *is* bounded) or on the instance (a set with no constraints *is not* bounded). This is what `isbounded(P)` and `isboundedtype(P)` was supposed to differentiate.